go.mod

```diff
@@ -23,6 +24,7 @@ require (
 	github.com/pion/webrtc/v3 v3.0.32
 	github.com/pkg/browser v0.0.0-20180916011732-0a3d74bf9ce4
 	github.com/rjeczalik/notify v0.9.2
+	github.com/spf13/afero v1.6.0
```

Comment:
Is this dependency still necessary, now that Go has a built-in in-memory FS for testing? https://pkg.go.dev/testing/fstest

Reply:
Unfortunately io/fs only supports read-only right now. But there's a [proposal in the works](https://github.com/golang/go/issues/45757) to support a writeable filesystem interface!